source/chapters/user_interface.rst

```diff
@@ -79,6 +79,26 @@ Track Information Display
             to the speed you are tapping, go to :ref:`control-keyboard` for more
             information.
 
+**Cover Art**
+  The cover art of the track loaded into the deck is displayed on the left side of the track. This is the same as the cover art that is shown under the :guilabel:`Cover Art` column heading in the Mixxx library. 
+  In Deere skin you can choose to enable or disable the cover art by clicking the gear icon in the upper right corner, then toggling the :menuselection:`Decks --> Cover Art` option. Clicking on the cover art will expand it into a full blown image.
+
+**Key**
+  The key displays the current musical key of the loaded track after pitch shifting. The key is displayed below the BPM (Tempo). This is the same as the key that is shown under the :guilabel:`Key` column heading in the Mixxx library. If it is hidden, you can drag the horizontal scroll bar at the bottom of the Mixxx library section, to the right, and the column will show. 
+  Clicking the key will expand it to reveal more options: 
+
+  .. figure:: ../_static/2.2/key_options.png
```

Comment:
The Match button is cropped. While this is a bug when Mixxx is at minimal size, please make the Mixxx window wider and take another screenshot.

Reply:
Okay, On it!

Reply:
Hi  @ronso0 here is a full screenshot from which i cropped the match button. My Mixxx window is maximized and taking up the entire screen but the Match button still displays as cropped.

![image](https://user-images.githubusercontent.com/28389950/96809280-3c1ee280-1423-11eb-9271-280a9209c7c2.png)

I also tried toggling off the some of the options in the Mixer settings to create more space so that it could display fully but it still didn't work. Maybe it's a Windows OS problem...? Is there another way i could get it to display fully?

Reply:
Alright then.